inst/rmarkdown/templates/ieee_article/skeleton/skeleton.Rmd

```diff
@@ -66,9 +66,9 @@ abstract: |
   The abstract goes here.
   On multiple lines eventually.
 
+bibtex: true
```

Comment:
What does this option do?

Reply:
sorry, typo, should be `biblatex`... i inserted it because i wanted that the template uses biblatex and the `citation_package = "biblatex"` does not produce the correct references..
might be worth having another look..